### ISSUE TITLE
Xcode complains about missing keychain-access-groups entitlement

### ISSUE THREAD

**Issue by bjarkehs** [square/Valet#45]
Hi,

I'm having some issues exporting an app that has a framework included that uses Valet. It only uses VALValet, so it shouldn't require shared keychain as far as I'm concerned. Is there anything I'm missing? Xcode says:
```
No matching provisioning profiles found for "App"
None of the valid provisioning profiles allowed the specified entitlements: application-identifier, keychain-access-groups.
```

**Comment by dfed** [comment]
Valet did not cause this problem, since Valet does not change your entitlements. You're likely seeing this error because you've misconfigured your entitlements under Project -> Capabilities.

I recommend using google / stack overflow to find the answer to this. Closing this out.

FWIW, a quick google search revealed this, which may or may not be helpful:
http://stackoverflow.com/questions/28006773/xcode6-error-no-matching-provisioning-profiles-found-for-application

**Comment by bjarkehs** [comment]
It seems to have been a problem with how I included Valet. :) It all works now after tweaking my included Framework. Thanks for making Valet, it's amazing :+1:

**Comment by dfed** [comment]
Glad you got it working! I take it you are using git submodules to bring in Valet? Out of curiosity (and to help the next guy who hits this), what was the fix?

**Comment by bjarkehs** [comment]
Yeah, I couldn't figure out a way to get Cocoapods into my Framework project without having it be Cocoapods itself (which isn't going to work due to having to provide precompiled frameworks). So I had it included in my project, as described in the README.

Granted my fix might also be tied to the project I included the framework in. But it was a very simple project with nothing with keychain in it. So the required entitlement is very likely to have come from the Framework.

I included the Valet project in my framework project. Then I went to Build Phases and added Valet.framework (iOS) under Link Binary With Libraries, then added Valet-iOS (Valet) to Target Dependencies. Then I could use my framework in my actual app with Valet added through Cocoapods.

This may all sound super simple, but I had the issues when I used the .a instead of the .framework.
